ql/src/java/org/apache/hadoop/hive/ql/exec/repl/incremental/IncrementalLoadTasksBuilder.java

```diff
@@ -70,9 +70,10 @@
   private final HiveConf conf;
   private final ReplLogger replLogger;
   private static long numIteration;
+  private Long eventTo;
```

Comment:
Why do we need to store eventTo here? ReplLoadWork already have it and build method takes work as input argument.

Reply:
its not stored in ReplLoadWork ..passed to IncrementalLoadTasksBuilder

Reply:
yeah right...